src/3/world1.md

```diff
@@ -410,178 +411,180 @@ My body, which has no unnecessary meat, is in fact the best!"
 
 "What is wrong with being flat!?"
 
-_TL Note:_ Oi, Rinne did you get bulling license on our flat goddess!?
+_TL Note:_ Oi, Rinne did you get bullying license on our flat goddess!?
+_ED Note:_ She's not wrong though... :eyes:
```

Comment:
Remember to add empty lines to separate sentence by empty line, when it is rendered

Reply:
! Forgot! >w< Thanks!